HelpSource/Guides/Internal-Snooping.schelp

```diff
@@ -5,27 +5,36 @@ categories:: Debugging
 You can inspect much of the internal structure of the class library and other data structures.
 This can often be useful for research and debugging purposes.
 
-warning:: The keyboard shortcuts mentioned here only applies to the Mac SC.app. This document should probably be updated. ::
+note:: The keyboard shortcuts mentioned here only applies to the standard supecollider IDE.::
 
 section:: Class Definitions, Implementations, and References
 
-Selecting the name of any Class (e.g. Object) and typing cmd-j will open its class definition file.
+subsection::cmd-i
+Selecting the name of any Class (e.g. Object) and typing strong::cmd-i:: will open its class definition file.
 
-note:: Any keypresses that open files identify the files using SuperCollider strings, which may contain only ASCII characters up to code point 127. If any part of the path to your SuperCollider installation contains character codes greater than 127 (such as the old Mac convention of identifying program folders with option-F), automatic file opening will fail. ::
 
-Selecting the name of any method (e.g. play) and typing cmd-y will open a window showing all implementations of that method and their arguments. Selecting one of those classes and methods (e.g. Sample:play) and typing cmd-j will open the class definition at that method. (Note that cmd-y only shows implementations, and does not indicate inheritance).
+subsection::cmd-u
+Selecting the name of any method (e.g. play) and typing strong::cmd-u:: will open a window showing all implementations of that method and their arguments. Selecting one of those classes and methods (e.g. Sample:play) and typing cmd-j will open the class definition at that method. (Note that cmd-y only shows implementations, and does not indicate inheritance).
```

Comment:
hi, 
why not explain all these based on the Language Menu entries, which should be cross-platform?
Then mention they have shortcuts (shown in the Menu) which are real timesavers worth learning,
and point to Reference/KeyboardShortcuts for the full overview / comparison of shortcuts in use.